packages/protocol/migrations/07_reserve_spender_multisig.ts

```diff
@@ -14,7 +11,8 @@ const initializeArgs = async (): Promise<any[]> => {
   ]
 }
 
-module.exports = deploymentForProxiedContract<ReserveSpenderMultiSigInstance>(
+// TODO @amyslawson has to register address for contractKit -- see if this is necessary
```

Comment:
Ah, indeed 😄 

I think a good goal would be to avoid registering this contract. We could instead look the address up via the reserve contract whenever we need to use it, e.g. in CLI tools or contractkit tests